When I try to `./autogen.sh --quick' CVS gettext, using git Autoconf,
things fall over like this (several instances):

| gnulib-m4/openmp.m4:29: error: m4_copy: won't overwrite defined macro: 
_AC_LANG_OPENMP(C++)
| gnulib-m4/openmp.m4:29: the top level
| autom4te: /usr/bin/m4 failed with exit status: 1
| aclocal: autom4te failed with exit status: 1

I'm not quite sure whether an Autoconf bug/incompatibility is involved
here (therefore the Cc: to bug-autoconf).  m4_copy wasn't documented in
earlier Autoconf releases, but there may still have been problematic
semantic changes, I don't know.

Anyway, the following patch seems to fix it.  OK to apply?

Cheers,
Ralf

2008-12-03  Ralf Wildenhues  <[EMAIL PROTECTED]>

        * m4/openmp.m4 (AC_OPENMP): Do not define with Autoconf 2.62 or
        newer.

Upstream-Status: Pending

Index: gettext-0.17/gettext-tools/gnulib-m4/openmp.m4
===================================================================
--- gettext-0.17.orig/gettext-tools/gnulib-m4/openmp.m4	2010-02-01 10:51:00.437788710 +0000
+++ gettext-0.17/gettext-tools/gnulib-m4/openmp.m4	2010-02-01 10:51:50.697789220 +0000
@@ -1,11 +1,13 @@
-# openmp.m4 serial 4
-dnl Copyright (C) 2006-2007 Free Software Foundation, Inc.
+# openmp.m4 serial 5
+dnl Copyright (C) 2006-2008 Free Software Foundation, Inc.
 dnl This file is free software; the Free Software Foundation
 dnl gives unlimited permission to copy and/or distribute it,
 dnl with or without modifications, as long as this notice is preserved.
 
 dnl This file can be removed once we assume autoconf >= 2.62.
 
+m4_version_prereq([2.62],, [
+
 # _AC_LANG_OPENMP
 # ---------------
 # Expands to some language dependent source code for testing the presence of
@@ -90,3 +92,5 @@
   fi
   AC_SUBST([OPENMP_]_AC_LANG_PREFIX[FLAGS])
 ])
+
+])
